.gitignore

```diff
@@ -1,6 +1,6 @@
 // Our automatic versioning scheme generates the following file
 // NEVER put it in the repository
-_Version.h
+Marlin/_Version.h
```

Comment:
I think i was under the impression that it wouldn't ignore _Version.h unless I specified the directory it was in. All fixed!